transcription/transcription.go

```diff
@@ -29,3 +32,32 @@ func msgHeaders(from string, to []string, subject string) string {
 	msgHeaders := []string{fromHeader, toHeader, subjectHeader}
 	return strings.Join(msgHeaders, "\r\n")
 }
+
+// DownloadFileFromURL locally downloads an audio file stored at url.
+func DownloadFileFromURL(url string) error {
+	// Taken from https://github.com/thbar/golang-playground/blob/master/download-files.go
+	tokens := strings.Split(url, "/")
+	fileName := tokens[len(tokens)-1]
+
+	// Create the file
```

Comment:
Make sure to remove these unnecessary comments. Opt for self-documenting code instead!